testsuite/compat/src/test/java/org/jboss/as/test/compat/jpa/hibernate/transformer/VerifyHibernate51CompatibilityJDSEnabledTransformerTestCase.java

```diff
@@ -30,12 +30,16 @@
 import org.jboss.shrinkwrap.api.spec.EnterpriseArchive;
 import org.jboss.shrinkwrap.api.spec.WebArchive;
 import org.junit.BeforeClass;
+import org.junit.Ignore;
 import org.junit.runner.RunWith;
 
 /**
  * Enable Hibernate bytecode transformer for application with jboss-deployment-structure.xml
  */
 @RunWith(Arquillian.class)
+@Ignore // WFLY-11991 added a StateType#internalNullSafeGet that is not supported by default
```

Comment:
instead of ignoring test and I think it would be better to just set required property. https://github.com/scottmarlow/wildfly/pull/29 does it

Reply:
Thanks @simkam merged and squashed your change!